get-started/versioning-availability.md

```diff
@@ -7,208 +7,53 @@ products:
 description: Learn how Elastic handles versioning and feature availability in the docs. Find the product versions that are supported, how to read availability badges, and...
 ---
 
-# Versioning and availability in Elastic Docs
+# Versioning and availability
 
-Learn how Elastic Docs handles versioning, feature availability, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the Elastic Stack versioning and confidently navigate our continuously updated documentation.
+Learn how Elastic products are versioned, the lifecycle of features, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the versioning and confidently navigate our continuously updated documentation.
 
-## Frequently asked questions
+## Understanding versioning
 
-### Where can I find documentation for the latest version of the {{stack}}?
+Each deployment type has a unique versioning strategy. For example, Serverless and ECH have releases that are distinguished by their date. Because Serverless is automatically updated, you always have the latest updates.
```

Comment:
> Serverless and ECH have releases that are distinguished by their date.

Is that necessary to say? I think this adds more confusion than clarity.

We could rather say that: 
- Most of Elastic products share the same versioning pattern, based on the Elastic Stack: Elasticsearch, Kibana, ...
- Orchestrators such as ECE and ECK, and other tools such as code clients and SDKs, are versioned independently of the Elastic Stack, but the pattern follows the same principles (X.Y.Z)
- The Elastic Cloud console and Serverless projects are always automatically updated with the latest updates

Reply:
Edited!